src/models/task.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require("mongoose");
+const {Schema} = mongoose;
+
+const taskScheme = new Schema({
+    text: String,
+    isCheck: Boolean
```

Comment:
create isCheck on the back by default, not on the front. Then it will not be necessary to validate the checkbox when creating a task